Cornelica/scenario/007_06.txt

```diff
@@ -10,6 +10,16 @@
 
 @bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
 
+[Lily]"This is the brothel..."
+
+@bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
+
+[Lily]"... or, where it was..."
+
+@bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
+
+[Lily]"but the male escorts quickly became impotent... \n...so it went bankrupt."
+
```

Comment:
o.O - Are these added by mistake or something? They're the same as right below these lines.

Reply:
grrr.... that file is screw. I can't figure it out. Right before those is had like
<<<<<<<<<<HEAD
stuff... i will try to re-fixit, but if keep fucking up i can send you the correct file so you can fix directly into the code